lib/spack/spack/detection/path.py

```diff
@@ -234,7 +234,6 @@ def by_executable(packages_to_check, path_hints=None):
                 exe_pattern_to_pkgs[exe].append(pkg)
         # Add Windows specific, package related paths to the search paths
         path_hints.extend(compute_windows_program_path_for_package(pkg))
-
```

Comment:
This looks vestigial (no modifiers to this file other than whitespace)

Reply:
Probably leftover damage from some debug output; fixed now